index.js

```diff
@@ -116,6 +122,29 @@ function compileTS(tsconfigFile, projectPath, tsinclude) {
   });
 }
 
+function mungeCompilerOptions(compilerOptions) {
+  let optionDefaults = {
+    lib: []
+  };
+  let optionOverrides = {
+    target: 'es2017',
+    module: 'es2015',
+    outDir: 'dist'
+  };
+
+  compilerOptions = Object.assign({}, optionDefaults, compilerOptions, optionOverrides);
```

Comment:
This will override `tsconfig.compilerOptions` for the options in `optionOverrides`. Please advise if there is a preferred approach to generating the desired output structure while accepting conflicting options.